src/plugins/foxPage/hooks/useLpApr.ts

```diff
@@ -0,0 +1,51 @@
+import { Fetcher, Token } from '@uniswap/sdk'
+import IUniswapV2Pair from '@uniswap/v2-core/build/IUniswapV2Pair.json'
+import { useEffect, useState } from 'react'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+
+import {
+  FOX_TOKEN_CONTRACT_ADDRESS,
+  UNISWAP_V2_WETH_FOX_POOL_ADDRESS,
+  WETH_TOKEN_CONTRACT_ADDRESS,
+} from '../const'
+import { calculateAPRFromToken0, getEthersProvider } from '../utils'
+import { useContract } from './useContract'
+import { useCurrentBlockNumber } from './useCurrentBlockNumber'
+
+export const useLpApr = () => {
+  const ethersProvider = getEthersProvider()
+  const [lpApr, setLpApr] = useState<string | null>(null)
+  const [loaded, setLoaded] = useState(false)
+  const blockNumber = useCurrentBlockNumber()
+
+  const liquidityContractAddress = UNISWAP_V2_WETH_FOX_POOL_ADDRESS
```

Comment:
I guess we could make this one programmatic, but given ETH/FOX on UniV2 is the only LP APR we will need, it's probably safer not to. It makes things more contained, and makes sure we do not use this hook for fetching anything else.